SPECS/perl/perl.spec

```diff
@@ -457,7 +526,7 @@ Requires:       perl-libs = %{perl_epoch}:%{perl_version}-%{release}
 %if %{defined perl_bootstrap}
 %gendep_perl_utils
 %endif
-
+Conflicts:      perl < 4:5.22.0-351
```

Comment:
I think we can also remove this one.

Reply:
There are quite a few other Conflicts in the SPEC - I think we should either keep all or remove all of them

Reply:
I'd just remove them. We don't have these other packages/older version the conflicts mention. Or at least that was the case with the ones I've taken a look at. I'd only keep any conflicts between packages from the latest version but I think I've never seen anything like that.

Reply:
Makes sense, I'll remove them all